host/python/src/one_sdk/__init__.py

```diff
@@ -0,0 +1,2 @@
+from one_sdk.client import OneClient
```

Comment:
This is first place I encountered the name of the package, so it is why here.

Since in Node.js is called @superface/one-sdk I assumed we will use one-sdk. Also when I scanned projects their seems to use - in name over _. https://pypi.org/search/?c=Programming+Language+%3A%3A+Python+%3A%3A+3

If I understand packaging correctly the folder with package in src should have name of the package.

Reply:
Unfortunately that doesn't seem to be valid. Module names have to be valid identifiers.

I looked at source of some of the packages in pypi and even though they are listed as `package-name` in `pyproject.toml`, in their `src` folder the modules are using `package_name`.

See for example
* https://pypi.org/project/in-place/ vs. https://github.com/jwodder/inplace/tree/master/src
* https://pypi.org/project/django-redis/ vs. https://github.com/jazzband/django-redis (no src, the module is in root of the repo)
* https://pypi.org/project/napari-czann-segment/ vs. https://github.com/sebi06/napari-czann-segment/tree/main/src

Reply:
Ok